### DISCUSSION TITLE
v0.1.9-alpha

### DISCUSSION THREAD

**Discussion by ZhengLinLei** [ZhengLinLei/stackblock.io#95] [Ideas]
Changelogs
```log
v0.1.9 -> 05/12/2023 -----------------------------------> TAG: v0.1.9
    - Fixed funding config
    - Added secret section
    - Added experimental bg music

    !TODO: Pending to finish bg music branch
    !TODO: Pending to finish Two function animation for v0.1.10    ------------------------> Deadlock
    !TODO: Pending to create context save for v0.1.10   -----------------------------------> Deadlock

```

## What's Changed
* Fixed funding.yml by @ZhengLinLei in https://github.com/ZhengLinLei/stackblock.io/pull/88
* Added secred section for achievements by @ZhengLinLei in https://github.com/ZhengLinLei/stackblock.io/pull/90
* Feature task add bg music by @ZhengLinLei in https://github.com/ZhengLinLei/stackblock.io/pull/66
* v0.1.9 stash by @ZhengLinLei in https://github.com/ZhengLinLei/stackblock.io/pull/93
* v0.1.9 Release by @ZhengLinLei in https://github.com/ZhengLinLei/stackblock.io/pull/94

**Full Changelog**: https://github.com/ZhengLinLei/stackblock.io/compare/v0.1.8...v0.1.9-alpha

<hr /><em>This discussion was created from the release <a href='https://github.com/ZhengLinLei/stackblock.io/releases/tag/v0.1.9-alpha'>v0.1.9-alpha</a>.</em>

**Comment by achechi15** [comment]
In what version are you planning to launch deadlock mode?

**Reply by ZhengLinLei** [reply]
In any of the version below. The integration is going to be partial.
